examples/onoff_light/src/main.rs

```diff
@@ -28,40 +31,39 @@ use rs_matter::data_model::objects::*;
 use rs_matter::data_model::root_endpoint;
 use rs_matter::data_model::system_model::descriptor;
 use rs_matter::error::Error;
-use rs_matter::mdns::{MdnsRunBuffers, MdnsService};
+use rs_matter::mdns::builtin::{
+    MDNS_IPV4_BROADCAST_ADDR, MDNS_IPV6_BROADCAST_ADDR, MDNS_SOCKET_BIND_ADDR,
+};
+use rs_matter::mdns::MdnsService;
+use rs_matter::persist::Psm;
 use rs_matter::secure_channel::spake2p::VerifierData;
-use rs_matter::transport::core::RunBuffers;
-use rs_matter::transport::network::{Ipv4Addr, Ipv6Addr, NetworkStack};
+use rs_matter::transport::core::{PacketBuffers, MATTER_SOCKET_BIND_ADDR};
+use rs_matter::transport::network::{Ipv4Addr, Ipv6Addr, UdpBuffers};
 use rs_matter::utils::select::EitherUnwrap;
+use rs_matter::MATTER_PORT;
 
 mod dev_att;
 
-#[cfg(feature = "std")]
 fn main() -> Result<(), Error> {
     let thread = std::thread::Builder::new()
-        .stack_size(160 * 1024)
+        .stack_size(180 * 1024)
```

Comment:
is there some way to determine what a correct stack size should be? Maybe some comments explaining would help: here you seem to say 160 was not enough but 180 is ... how did we figure it out? Increase until it works (+ some buffer space) or something else?

Reply:
For now it is - unfortunately - and really - "increase until it works".
The required RAM currently varies between 140 to 180KB depending on machine architecture (i.e. 64 bit vs 32 bit) optimization settings (though you should really not try to run this with opt-level < 2 in production).

We need to lower this consumption of course, but I haven't got there yet. One easy way is always to just reduce the number of supported sessions, and simultaneous exchanges, but there are optimizations we should try before that.

Anyway, I'll put a comment on top.

Reply:
Done.